modules/UI/videolayout/SmallVideo.js

```diff
@@ -542,35 +539,42 @@ SmallVideo.prototype.showRaisedHandIndicator = function (show) {
         return;
     }
 
-    var indicatorSpanId = "raisehandindicator";
-    var indicatorSpan = this.getIndicatorSpan(indicatorSpanId);
-
-    indicatorSpan.innerHTML
-        = "<i id='indicatoricon' class='icon-raised-hand'></i>";
-
-    // adds a tooltip
-    UIUtil.setTooltip(indicatorSpan, "raisedHand", "top");
-    APP.translation.translateElement($(indicatorSpan));
+    var indicatorSpan = this.getIndicatorSpan({
+        id: 'raisehandindicator',
+        content: '<i id="indicatoricon" class="icon-raised-hand"></i>',
+        tooltip: 'raisedHand'
+    });
 
-    $(indicatorSpan).css("visibility", show ? "visible" : "hidden");
+    $(indicatorSpan)[show ? "show" : "hide"]();
```

Comment:
Why we are using jquery's show/hide methods instead of setting css properties? I was thinking that setting the css property is more efficient isn't it?